images/macos/provision/core/pipx-packages.sh

```diff
@@ -0,0 +1,25 @@
+source ~/utils/utils.sh
+
+export PATH="$PATH:/opt/pipx_bin"
+
+toolset=get_toolset_path
+pipx_packages=$(jq -r ".pipx[] .package" $toolset)
+
+for package in $pipx_packages; do
+    python_version=$(jq -r ".pipx[] | select(.package == \"$package\") .python" $toolset)
+    if [ "$python_version" != "null" ]; then
+        python_path="/opt/hostedtoolcache/Python/$python_version*/x64/bin/python$python_version"
```

Comment:
We don't have `/opt/hostedtoolcache` on MacOS

Reply:
Fixed it